raw_data_processor/src/main/scala/scalalab3/lyricsengine/reader/Reader.scala

```diff
@@ -0,0 +1,49 @@
+package scalalab3.lyricsengine.reader
+
+import com.typesafe.config.ConfigFactory
+
+import scala.io.Source
+
+object Reader {
+  val conf = ConfigFactory.load();
+
+  def main(args: Array[String]) {
+    print(extract())
+  }
+
+  def extract(): MSD = {
+    val path = conf.getString("path_to_msd")
```

Comment:
I'd suggest to separate config part from execution part. It will give a bit more flexibility.